doc/source/whatsnew/v0.20.0.txt

```diff
@@ -1113,3 +1113,4 @@ Other
 - Compat for 32-bit platforms for ``.qcut/cut``; bins will now be ``int64`` dtype (:issue:`14866`)
 - Bug in interactions with ``Qt`` when a ``QtApplication`` already exists (:issue:`14372`)
 - Avoid use of ``np.finfo()`` during ``import pandas`` removed to mitigate deadlock on Python GIL misuse (:issue:`14641`)
```

Comment:
to the numeric section of bug fixes. I thin this is ok as a bug fix, though if you think as a user it would warrant a bigger mention, a sub-section would be ok too.